src/jobs/summary-notification.ts

```diff
@@ -0,0 +1,69 @@
+ * Copyright (c) 2021 Hyperjump Technology                                        *
+ *                                                                                *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+
+import { hostname } from 'os'
+import format from 'date-fns/format'
+import { getConfig } from '../components/config'
+import { getSummary } from '../components/logger/history'
+import { sendNotifications } from '../components/notification'
+import getIp from '../utils/ip'
+import { log } from '../utils/pino'
+import { publicIpAddress } from '../utils/public-ip'
+
+export async function getSummaryAndSendNotif() {
+  const config = getConfig()
+  const { notifications } = config
+
+  if (!notifications) return
+
+  try {
+    const summary = await getSummary()
+
+    sendNotifications(notifications, {
+      subject: `Monika Status`,
+      body: `Status Update ${format(new Date(), 'yyyy-MM-dd HH:mm:ss XXX')}
+              Host: ${hostname()} (${[publicIpAddress, getIp()]
+        .filter(Boolean)
+        .join('/')})
+              Number of probes: ${summary.numberOfProbes}
+              Average response time: ${
+                summary.averageResponseTime
+              } ms in the last 24 hours
+              Incidents: ${summary.numberOfIncidents} in the last 24 hours
+              Recoveries: ${summary.numberOfRecoveries} in the last 24 hours
+              Notifications: ${summary.numberOfSentNotifications}`,
```

Comment:
The whitespaces at the beginning of lines need to be removed

<img width="905" alt="Screen Shot 2021-09-17 at 10 25 21" src="https://user-images.githubusercontent.com/22452017/133719418-8a72cc53-8182-4a6d-b827-71b5cf50140b.png">

Reply:
Nice catch, thank you. Updated here [1246deb](https://github.com/hyperjumptech/monika/pull/417/commits/1246deba2064504ca3dc0764a4234c792c6f80f9).